designer/client/button-dropdown.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+
+export default class buttonDropdown extends React.Component {
```

Comment:
Would this ideally be ButtonDropdown? It seems like other classes are named with capitals at the start

Reply:
that's my bad -- I was lazy with the cherry-pick! I'll add an issue assigned to me